README.md

```diff
@@ -4,10 +4,15 @@
 
 * run `mvn clean test`
 
-## To see a report:
+## To access the report manually:
+1. Generate the report by running: `mvn allure:report`
+2. open `target/site/allure-maven-plugin/index.html`
```

Comment:
This way doesn't really work, at least in Chrome. 
Browser doesn't allow to open it as local file:

```
Access to XMLHttpRequest at 'file:///Users/andrei/projects/selenide-examples/selenide-allure-junit/target/site/allure-maven-plugin/widgets/behaviors.json' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https, isolated-app.
```

So the only possible way is `mvn allure:serve`.

Reply:
Indeed, It won't open even in Firefox due to **The Same Origin Policy** , unless we use an HTTP server like http-party/http-server](https://github.com/http-party/http-server)